src/main/java/com/suse/salt/netapi/calls/LocalCall.java

```diff
@@ -44,31 +44,36 @@ public class LocalCall<R> extends AbstractCall<R> {
     private final Optional<?> metadata;
     private final Optional<Integer> timeout;
     private final Optional<Integer> gatherJobTimeout;
+    private final Optional<List<String>> executors;
+    private final Optional<Map<String, ?>> executorOptions;
 
     public LocalCall(String functionName, Optional<List<?>> arg,
             Optional<Map<String, ?>> kwarg, TypeToken<R> returnType,
             Optional<?> metadata, Optional<Integer> timeout,
-            Optional<Integer> gatherJobTimeout) {
+            Optional<Integer> gatherJobTimeout, Optional<List<String>> executors,
```

Comment:
In this constructor,  you have enhanced it to add 2 new parameters for the executor. Would make sense to keep this constructor and create a new one with all the parameters? By changing it, everyone calling this constructor will need to change their code, even if they don't want the executors.

Reply:
sure